### PR TITLE
fixes #31, adds a popup letting people know about the github page

### LINKED ISSUE #31: Modal inviting comments / contributions
someone recommended that we include a modal or some form of info box that invites people to make comments / suggestions on how the website works. I think this is a great idea. We can include a link to this repo and a short message. thoughts?

### PR DESCRIPTION
**Pull request by codehag**
fixes #31

### CODE REVIEW

**Review by xtuc — APPROVED** [review]
I like it! Looks good on mobile too

File: _layouts/default.html [hunk 1]
```diff
@@ -30,5 +30,14 @@ <h1 class="page-title">Specifying JavaScript.</h1>
           </span>
       </section>
     </main>
+    <div class="popup">
+      <input type="checkbox" id="one" class="hidden" name="ossm">
+      <label for="one" class="alert-message">
+        <strong> <i class="fa fa-heart"></i> Attention</strong> This site is in BETA! We are
+        looking for feed back. Please add issues to our <a
+                    href="https://github.com/tc39/tc39-web-draft/issues" target="_blank">github page</a>!!
+        <span class="close">×</button>
```
Comment (Regaddi) [inline _layouts/default.html]: `<span>...</button>` doesn't seem right to me.
Please add a bit of padding to the button to make it clickable on touch devices.
The touch area is currently only 9px wide and 30px high.

File: _sass/_popup.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+}
+
+.fa{
+  line-height:30px;
+  font-size:30px;
+  display:inline-bloxk;
```
Comment (Regaddi) [inline _sass/_popup.scss]: typo `inline-bloxk`

File: _sass/_popup.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
+.popup {
```
Comment (Regaddi) [inline _sass/_popup.scss:1]: Seems like you haven't activated stylelint in your editor.
Please do a `npm install` and add a corresponding stylelint plugin to your editor.

For VSCode: https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint
For Atom: https://atom.io/packages/linter-stylelint
For SublimeText: https://github.com/SublimeLinter/SublimeLinter-stylelint

File: _layouts/default.html [hunk 4]
```diff
@@ -30,5 +30,14 @@ <h1 class="page-title">Specifying JavaScript.</h1>
           </span>
       </section>
     </main>
+    <div class="popup">
+      <input type="checkbox" id="one" class="hidden" name="ossm">
+      <label for="one" class="alert-message">
+        <strong> <i class="fa fa-heart"></i> Attention</strong> This site is in BETA! We are
```
Comment (Regaddi) [inline _layouts/default.html]: Can't find definition of `fa-heart`, so no heart is shown in the UI.

File: _sass/_popup.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+.fa{
+  line-height:30px;
+  font-size:30px;
+  display:inline-bloxk;
+  width:35px;
+  height:35px;
+}
+
+.hidden{
```
Comment (Regaddi) [inline _sass/_popup.scss]: This could potentially be moved to a utility file.

File: _sass/_popup.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+             moving 0.3s forwards ease-in-out;
+}
+.pointer-cursor{
+   cursor: pointer; cursor: hand;
+}
+.close{
+
+    float:right;
+    color: #b94b45;
+    cursor: pointer; cursor: hand;
```
Comment (Regaddi) [inline _sass/_popup.scss]: Code repetition of cursor-definitions.

**Review by Regaddi — CHANGES_REQUESTED** [review]

**Pull request merged**